suite-native/app/upload-to-firebase.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
+
+if [[ "$EAS_BUILD_PLATFORM" == "android" ]]; then
+  curl -sL https://firebase.tools | bash
+
+  commit_hash=$(git log -1 --pretty=format:"%H")
+  release_notes="Bug fixes and improvements. Last commit hash: $commit_hash"
+
+  firebase appdistribution:distribute /home/expo/workingdir/build/suite-native/app/android/app/build/outputs/apk/release/app-release.apk \
```

Comment:
Please don't use this absolute path, this could be changed without notice and it will break our build. Better use env variable `EAS_BUILD_WORKINGDIR` as starting point so it won't break if they change runners implementation.

https://docs.expo.dev/build-reference/variables/#built-in-environment-variables

Reply:
https://github.com/trezor/trezor-suite/pull/10898/commits/ffdda177dc654f11effb3b45beb83d4b6f476162

Reply:
Had to rebase, but fixed, just check the file please.